docs/promscale_alerting.md

```diff
@@ -0,0 +1,226 @@
+      component: connector
+    annotations:
+      summary: High error rate in querying Promscale
+      description: "Evaluating queries via Promscale has {{ $value }} error rate."
+  - alert: PromscaleQueryHighErrorRate
+    expr: |
+      (
+        sum by (job, instance, type) (
+          rate(promscale_query_requests_total{code=~"5.."}[5m])
+        )
+      /
+        sum by (job, instance, type) (
+          rate(promscale_query_requests_total[5m])
+        )
+      ) > 0.1
+    for: 1m
+    labels:
+      severity: critical
+      component: connector
+    annotations:
+      summary: High error rate in querying Promscale
+      description: "Evaluating queries via Promscale had {{ $value }} error rate."
+  - alert: PromscaleQueryHighLatency
+    expr: |
+      (
+        histogram_quantile(
+          0.90,
+          sum by (job, instance, type, le) (
+            rate(promscale_query_duration_seconds_bucket[5m])
+          )
+        ) > 5
+      and
+        sum by (job, instance, type) (
+          rate(promscale_query_duration_seconds_bucket[5m])
+        ) > 0
+      )
+    for: 1m
+    labels:
+      severity: warning
+      component: connector
+    annotations:
+      summary: Slow Promscale querying
+      description: "90% of the queries took {{ $value }} seconds to evaluate."
+  - alert: PromscaleQueryHighLatency
+    expr: |
+      (
+        histogram_quantile(
+          0.90,
+          sum by (job, instance, type, le) (
+            rate(promscale_query_duration_seconds_bucket[5m])
+          )
+        ) > 10
+      and
+        sum by (job, instance, type) (
+          rate(promscale_query_duration_seconds_bucket[5m])
+        ) > 0
+      )
+    for: 1m
+    labels:
+      severity: critical
+      component: connector
+    annotations:
+      summary: Slow Promscale querying
+      description: "90% of the queries took {{ $value }} seconds to evaluate."
+# Cache
+  - alert: PromscaleCacheHighNumberOfEvictions
+    expr: |
+      (
+        sum by (job, instance, name, type) (
+          rate(promscale_cache_evictions_total[5m])
+        )
+      /
+        sum by (job, instance, name, type) (
+          promscale_cache_capacity_elements
+        )
+      ) > 0.2
+    for: 5m
+    labels:
+      severity: warning
+      component: connector
+    annotations:
+      summary: High cache eviction in Promscale
+      description: "Promscale clockcache is evicting at {{ $value }} entries a second."
+  - alert: PromscaleCacheTooSmall
+    expr: |
+      (
+        sum by (job, instance, type, name) (
+          rate(promscale_cache_query_hits_total[5m])
+        )
+      /
+        sum by (job, instance, type, name) (
+          rate(promscale_cache_queries_total[5m])
+        )
+      ) < 0.9
+    for: 5m
+    labels:
+      severity: warning
+      component: connector
+    annotations:
+      summary: High cache eviction in Promscale
+      description: "Promscale clockcache has a hit ratio of {{ $value }}."
+```
```

Comment:
Putting this content in an `md` file allows users only to copy it manually without any ability to use this in an automated fashion. Instead, let's follow how other projects include such content by doing the following:
1. Create a directory structure as follows:
```
docs
└── mixin
    ├── alerts
    │   └── alerts.yaml
    ├── dashboards
    │   └── dashboard.yaml
    └── README.md
```
2. Put the content from the highlighted section in `alerts.yaml`
3. Put the instructions on how to use this in `README.md` so it is easily readable when going to the directory via GitHub UI

Such structure will allow us to easily extend content and in the future potentially include it in https://monitoring.mixins.dev/

Side note: This such directory schema will create a [monitoring mixin](https://github.com/monitoring-mixins/docs/blob/master/design.pdf).